concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2109,13 +2109,18 @@ jobs:
           run:
             path: sh
             args:
-            - -e
```

Comment:
Not knowing much about shellscript, I've removed this as it has caused the code to stop at [line 2117](https://github.com/alphagov/paas-cf/pull/930/files/a115a365b2540f6d5cf216565880b5de7ca26a81#diff-8f7658bb02a08dfb771f38694a7b075fR2117) and not continuing to the following if statement.

Regarding the `grep` and empty `vms.txt`:

```sh
→ mv vms.txt _vms.txt
→ touch vms.txt
→ ./test.sh
Success: All VMs are up and running.
```

I'd be more than happy, to use different solution, but I may need some more guidance.